src/main/java/dev/cammiescorner/arcanuscontinuum/common/entities/living/OpossumEntity.java

```diff
@@ -47,7 +47,8 @@
 import java.util.UUID;
 
 @SuppressWarnings("ALL")
-public class OpossumEntity extends TameableEntity implements SmartBrainOwner<OpossumEntity> {
+public class
+OpossumEntity extends TameableEntity implements SmartBrainOwner<OpossumEntity> {
```

Comment:
unnecessary formatting change

Reply:
That's kind of weird lol I didn't change the formatting. Might have happened by mistake.